dashboard/config/routes.rb

```diff
@@ -675,7 +675,8 @@
   # Routes used by standards info dialog
   post '/dashboardapi/v1/users/:user_id/set_standards_report_info_to_seen', to: 'api/v1/users#set_standards_report_info_to_seen'
 
-  post '/dashboardapi/v1/teacher_scores', to: 'api/v1/teacher_scores#score_stage_for_section'
+  post '/dashboardapi/v1/teacher_scores', to: 'api/v1/teacher_scores#score_stages_for_section'
+  post '/dashboardapi/v1/teacher_scores/get', to: 'api/v1/teacher_scores#get_teacher_scores_for_script'
```

Comment:
this looks like it should be a `GET` and not a `POST`, assuming it is retrieving data without modifying anything on the server

Reply:
I thought so too, but by my reading of these docs (https://edgeguides.rubyonrails.org/action_controller_overview.html#parameters)suggests that if you want to pass params not in the url in an HTTP request, you have to do it in a POST request.  " The second type of parameter is usually referred to as POST data. This information usually comes from an HTML form which has been filled in by the user. It's called POST data because **it can only be sent as part of an HTTP POST request**. "

Reply:
Good find! Is there a reason you "want to pass params not in the url", though? To make the GET request work, you can put any request params in the request URL.  Since you are just passing section_id and stage_id, these should be fine to put in the request URL because they are not very long. if you were needing to pass a larger amount of data, then it would make sense to convert it to a POST.